hub/core/tensor.py

```diff
@@ -648,6 +649,31 @@ def __iter__(self):
         for i in range(len(self)):
             yield self.__getitem__(i, is_iteration=True)
 
+    def get_full_point_cloud_numpy(
```

Comment:
are we sure about adding this method to tensor? seems point cloud specific and will also not raise exception on being used by other htypes